src/project-2.js

```diff
@@ -11,15 +12,35 @@ const greeting = (language) => {
   // language: 'Spanish' -> 'Hola!'
   // language: 'Chinese' -> 'Ni Hao!'
   // if language is undefined return 'Hello!'
+  switch (language) {
+    case 'German':
+      return 'Guten Tag!';
+    case 'Spanish':
+      return 'Hola!';
+    case 'Chinese':
+      return 'Ni Hao!';
+    default:
+      return 'Hello!';
+  }
 };
 
 const isTenOrFive = (num) => {
```

Comment:
Opportunity for refactor - One-liner:
```javascript
const isTenOrFive = (num) => num === 10 || num === 5 ? true : false;
```

Reply:
👍  And even simpler: `const isTenOrFive = num => (num === 10 || num === 5);`